### DISCUSSION TITLE
Allow mapping of GitHub Teams to roles in Grafana

### DISCUSSION THREAD

**Discussion by inductor** [grafana/grafana#50369] [Platform]
Taken over from https://github.com/grafana/grafana/issues/8875

Since no one seems to have created this discussion yet here I am :)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
